### PR TITLE
More command macros

### PR DESCRIPTION
**Pull request by PokeWorldBW**

### CODE REVIEW

File: scripts/mafia.js [hunk 1]
```diff
@@ -5135,9 +5167,17 @@ function Mafia(mafiachan) {
         return true;
     };
 
-    this.addPhaseStalkAction = function (user, action, target) {
+    this.addPhaseStalkAction = function (user, action, target, after, redir) {
+    	if (after !== "*" && after !== undefined) {
+    	    after = ":" + after
```
Comment (Fuzzysqurl) [inline scripts/mafia.js]: Missing semicolon.

File: scripts/mafia.js [hunk 2]
```diff
@@ -5135,9 +5167,17 @@ function Mafia(mafiachan) {
...
+    	}
+    	else after = "";
+    	if (redir !== "*" && redir !== undefined) {
+    	    redir = "@" + redir
```
Comment (Fuzzysqurl) [inline scripts/mafia.js]: Missing semicolon.

File: scripts/mafia.js [hunk 3]
```diff
@@ -5135,9 +5167,17 @@ function Mafia(mafiachan) {
...
+    	}
+    	else redir = "";
```
Comment (Fuzzysqurl) [inline scripts/mafia.js]: Expected "{" but instead saw "redir"

File: scripts/mafia.js [hunk 4]
```diff
@@ -5135,9 +5167,17 @@ function Mafia(mafiachan) {
         return true;
     };
 
-    this.addPhaseStalkAction = function (user, action, target) {
+    this.addPhaseStalkAction = function (user, action, target, after, redir) {
+    	if (after !== "*" && after !== undefined) {
+    	    after = ":" + after
+    	}
+    	else after = "";
```
Comment (Fuzzysqurl) [inline scripts/mafia.js]: Expected "{" but instead saw "after"

**Comment by Fuzzysqurl** [comment]
Closes #1258

**Comment by PokeWorldBW** [comment]
Wait, lol, Miki already closed the commit this overlapped with. The mafia checker change was different

**Pull request merged**